lms/static/js/commerce/views/receipt_view.js

```diff
@@ -19,11 +19,6 @@ var edx = edx || {};
             this.useEcommerceApi = this.ecommerceBasketId || this.ecommerceOrderNumber;
             _.bindAll(this, 'renderReceipt', 'renderError', 'getProviderData', 'renderProvider', 'getCourseData');
 
-            /* Mix non-conflicting functions from underscore.string (all but include, contains, and reverse) into
-             * the Underscore namespace.
-             */
-            _.mixin(_s.exports());
```

Comment:
@andy-armstrong @AlasdairSwan Including following code 
```
'js/commerce/views/receipt_view': {
    exports: 'edx.commerce.ReceiptView',
    deps: ['jquery', 'backbone', 'underscore', 'underscore.string'],
    init: function() {
        // Set global variables that the receipt code is expecting to be defined
        require([
            'underscore',
            'underscore.string'
        ], function (_, str) {
            window._ = _;
            window._s = str;
        });
    }
 },
```
in the file `lms/static/js/spec/main.js` doesn't help resolve the error (for js spec test):
```
Firefox 28.0.0 (Ubuntu 0.0.0) ERROR
  TypeError: _.str is undefined
  at /edx/app/edxapp/edx-platform/lms/static/js/commerce/views/receipt_view.js:28
```
while doing `_.mixin(_.str.exports());`

Is there is something we are missing?